app/common/lib/randomUtil.js

```diff
@@ -0,0 +1,5 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports.random = () => Math.random()
```

Comment:
I tried, but was unsuccessful ☹️  I [verified other folks have the same issue](https://stackoverflow.com/questions/17011765/stubbing-date-now-and-math-random) and then implemented it this way